packages/frontend/src/pages/share.vue

```diff
@@ -59,10 +59,24 @@ const visibleUsers = ref([] as Misskey.entities.UserDetailed[]);
 
 async function init() {
 	let noteText = '';
-	if (title.value) noteText += `[ ${title.value} ]\n`;
-	// Googleニュース対策
-	if (text?.startsWith(`${title.value}.\n`)) noteText += text.replace(`${title.value}.\n`, '');
-	else if (text && title.value !== text) noteText += `${text}\n`;
+	if (title.value) {
+		noteText += `[ ${title.value} ]\n`;
+
+		//#region add text to note text
+		const duplicateTextRegex = new RegExp(`^${RegExp.escape(title.value)}\\s*`, 'u');
+		if (text && duplicateTextRegex.test(text)) {
+			// For the Google app https://github.com/misskey-dev/misskey/issues/16224
+			noteText += text.replace(duplicateTextRegex, '');
+		} else if (text && title.value === text) {
+			// Nothing to do
```

Comment:
`startsWith` でも同じことが実現できるように思いますが、あえて使っていない理由はありますか？
```suggestion
		if (text?.startsWith(title.value)) {
			// For the Google app https://github.com/misskey-dev/misskey/issues/16224
			noteText += text.replace(title.value, '').trimStart();
```

Reply:
最初`\s+`のつもりだったので正規表現を使っていましたが、`\s*`に気分が変わったので正規表現から変えてませんでした